### PR TITLE
update css for monitoring plugin

### PR DESCRIPTION
**Pull request by m-kharbat**
screen shot 2017-01-23 at 15 05 10
screen shot 2017-01-23 at 15 04 43

### CODE REVIEW

**Comment by Autophagy** [comment]
tutut

File: app/styles/main.less [hunk 1]
```diff
@@ -5,11 +5,11 @@
 @import "fonts";
 
 @import "common";
-@import "light-theme-variables";
+// @import "light-theme-variables";
 
 // import the "dark-theme-variables" less file
 // to enable the dark theme
-// @import "dark-theme-variables";
+@import "dark-theme-variables";
```
Comment (Autophagy) [inline app/styles/main.less]: Do we want to be committing this into the feature branch?

Comment (m-kharbat) [inline app/styles/main.less]: no it got committed by mistake :/

**Comment by Autophagy** [comment]
screenshot

**Pull request merged**